charts/lago/values.yaml

```diff
@@ -321,32 +356,31 @@ job:
 
 minio:
   enabled: false
-  # replicas: 2
-  # fullnameOverride: "my-lago-minio"
-  # endpoint: "http://minio.lago.dev"
-  # nameOverride: "minio"
-  # resources:
-  #   requests:
-  #     memory: "512Mi"
-  #     cpu: "500m"
-  #   limits:
-  #     memory: "1Gi"
-  #     cpu: "1"
-  # persistence:
-  #   size: 10Gi
-  # ingress:
-  #   enabled: true
-  #   ingressClassName: nginx
-  #   labels: {}
-  #   annotations: {}
-  #   path: /
-  #   hosts:
-  #     - minio.lago.dev
-  #   tls: []
-  # Note : only the first one will be used
-  # buckets:
-  #   - name: my-lago-minio
-  #     policy: none
-  #     purge: false
-  #     versioning: false
-  #     objectlocking: false
+  replicas: 1
+  fullnameOverride: lago-minio
+  endpoint: "http://minio.lago.dev"
```

Comment:
I'm a surprised to see this url hardcoded but I don't know 🤷‍♂️ 
I would expect something like this https://github.com/getlago/lago-helm-charts/blob/main/charts/lago/values.yaml#L3-L6

Reply:
yes its the same, it has to be changed in the `values.yaml` file during the installation!